parcel-transformer-telefunc/transformer.ts

```diff
@@ -0,0 +1,32 @@
+import {Transformer} from '@parcel/plugin';
+import {dirname} from "path";
+
+const findPkg = require("find-pkg");
+
+//import {transformTelefuncFile} from "telefunc/transformer";
+const {transformTelefuncFile} = require("telefunc/transformer");
+
+export default new Transformer({
+    async transform({asset}) {
+
+        // Determine package path:
+        const packageJsonFile =await findPkg(dirname(asset.filePath));
```

Comment:
Instead of using `findPkg()`, doesn't Parcel provide the user's app root directory?

Reply:
https://parceljs.org/plugin-system/api/#PluginOptions => isn't `projectRoot` what we want?